contractcourt/channel_arbitrator.go

```diff
@@ -1468,10 +1472,24 @@ func (c *ChannelArbitrator) findCommitmentDeadlineAndValue(heightHint uint32,
 		}
 
 		value := htlc.Amt.ToSatoshis()
-		totalValue += value
 
-		if htlc.RefundTimeout < deadlineMinHeight {
-			deadlineMinHeight = htlc.RefundTimeout
+		// Find the expiry height for this outgoing HTLC's incoming
+		// HTLC.
+		deadlineOpt := c.cfg.FindOutgoingHTLCDeadline(htlc)
+
+		// The deadline is default to the current deadlineMinHeight,
+		// and it's overwritten when it's not none.
+		deadline := deadlineMinHeight
+		deadlineOpt.WhenSome(func(d int32) {
+			deadline = uint32(d)
+
+			// We only consider the value is under protection when
+			// it's time-sensitive.
+			totalValue += value
```

Comment:
Why shouldn't the total value also include this value if it isn't under a deadline? Won't they all be offered to the sweeper with a set budget, witch'll be used as fee bumping resource for the inputs it's bundled with?

Reply:
yea this HTLC will also be offered to the sweeper, but we can be less aggressive here. The idea is, we want to CPFP the force close tx, because there are HTLCs that will timeout, so it makes sense to "borrow" some values from these HTLCs to accelerate the force close tx. However, if the HTLC is not time-sensitive, for instance, an outgoing htlc from the first hop, then there's no need to accelerate the FC tx, or, there's no need to ask this HTLC to "contribute" urgentness to the FC process.